examples/v1beta1/trial-images/enas-cnn-cifar10/Dockerfile.gpu

```diff
@@ -1,6 +1,6 @@
-# tensorflow-gpu=2.9.1, cuda=11.7.1
-# Ref: https://docs.nvidia.com/deeplearning/frameworks/tensorflow-release-notes/rel_22-08.html#rel_22-08
-FROM nvcr.io/nvidia/tensorflow:22.08-tf2-py3
+# tensorflow-gpu=2.10.1, cuda=11.8.0
+# Ref: https://docs.nvidia.com/deeplearning/frameworks/tensorflow-release-notes/rel-22-12.html#rel-22-12
+FROM nvcr.io/nvidia/tensorflow:22.12-tf2-py3
```

Comment:
I'm not sure if it works, but is is possible to change the base image `BASE_IMAGE_TENSORFLOW_GPU = "docker.io/tensorflow/tensorflow:2.11.0-gpu"`, here too?

Reply:
The `docker.io/tensorflow/tensorflow:2.11.0-gpu` is only provided for the linux/amd64. So we need to use `nvcr.io/nvidia/tensorflow:22.12-tf2-py3` to support linux/amd64 and linux/arcm64.

Reply:
I see now :)
Could you comment the reason in this Dockerfile too?
Since it would be hard to find the reason why the base trial image is `docker.io/tensorflow/tensorflow:2.11.0-gpu` in the katib trial source code, but this image uses another base image

Reply:
> Could you comment the reason in this Dockerfile too?

It makes sense. Thanks for the suggestion :)

Reply:
Done.